test/stdlib/tgmath.swift.gyb

```diff
@@ -189,8 +189,8 @@ extension ${T}: TGMath {
 
 MathTests.test("${T}") {
   ${T}.allTests()
-% if T == 'Double':
-  //  Functions that are defined only for Double
+% if T in ['Double','CGFloat']:
+  //  Functions that are defined only for Double and CGFloat
```

Comment:
This does run on 32-bit platforms, you know…

Reply:
Yes. These interfaces are provided on 32b too.

Reply:
Ah, the concern was that the results wouldn't match the tolerance.

Reply:
No worries there; on 64b you get the Double implementation, so if it passes for Double it passes for CGFloat. On 32b, it's implemented by calling Double, then rounding to CGFloat, so if the Double implementation passed, the error of CGFloat is like < 0.50000003 ulp at *worst*.